basic_gdeploy_doc.md

```diff
@@ -0,0 +1,298 @@
+#### Introduction of gdeploy and how to use gdeploy to configure glusterfs
+-----
+
+This document is intended to give a basic overview of the gdeploy tool like,
+what is gdeploy, what does it do, and how you can use it.
+
+##### What is glusterfs?
+
+GlusterFS is a scale-out network-attached storage file system. It has found applications 
+including cloud computing, streaming media services, and content delivery networks. GlusterFS 
+was developed originally by Gluster, Inc., then by Red Hat, Inc., after their purchase of 
+Gluster in 2011.
```

Comment:
> GlusterFS is a scalable network filesystem suitable for data-intensive tasks such as cloud storage and media streaming. GlusterFS is free and open source software and can utilize common off-the-shelf hardware. To learn more, please see the Gluster project home page.

from https://gluster.readthedocs.org/en/latest/ is a reasonably succinct introduction